kcidb/templates/build.j2

```diff
@@ -24,6 +24,43 @@
     -}}
 {% endmacro %}
 
+{%- macro status(architecture,
+                invalid_build_count,
+                valid_build_count,
+                unknown_build_count,
+                build_fmt) -%}
+   {{- build_fmt | format("?" if architecture is none
+                           else architecture,
+                           "❌" if invalid_build_count else "➖",
+                           (invalid_build_count | string)
+                           if invalid_build_count else "",
+                           "✅" if valid_build_count else "➖",
+                           (valid_build_count | string)
+                           if valid_build_count else "",
+                           "❓" if unknown_build_count else "➖",
+                           (unknown_build_count | string)
+                           if unknown_build_count else "") -}}
+{% endmacro %}
+
+{%- macro build_status(container, space="") -%}
```

Comment:
I think this should be named something like `container_status`, since this is producing a build status summary for a container, not one particular build. Besides the file is already called `build.j2` and having "build" in the name would be repeating ourselves.